brave/brave-core
# Update help link to be relevant for Windows users

Fixes https://github.com/brave/brave-browser/issues/11746

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Be on Windows
2. Have a fully built version
3. Try to do an update - update must fail
4. Verify link shown under `Learn more` goes to correct place

Would be great if there was a debug way to show the full status - but I find anything unfortunately

I tested myself on Windows by applying this patch manually:
```diff
diff --git a/patches/chrome-browser-resources-settings-about_page-about_page.html.patch b/patches/chrome-browser-resources-settings-about_page-about_page.h ml.patch
index 5d1e21798d..3a4d208479 100644
--- a/patches/chrome-browser-resources-settings-about_page-about_page.html.patch
+++ b/patches/chrome-browser-resources-settings-about_page-about_page.html.patch
@@ -1,13 +1,29 @@
 diff --git a/chrome/browser/resources/settings/about_page/about_page.html b/chrome/browser/resources/settings/about_page/about_page.html
-index e8299dcfeb8bda338290bd0f89811ca2e613be9f..888ce6e999104bb3ec2ae187213badde20d61be7 100644
+index e8299dcfeb8bda338290bd0f89811ca2e613be9f..ba9e1ba66b44388add93d47a9d853d2358cdd8e3 100644
 --- a/chrome/browser/resources/settings/about_page/about_page.html
 +++ b/chrome/browser/resources/settings/about_page/about_page.html
-@@ -72,9 +72,10 @@
+@@ -57,7 +57,7 @@
+           when update is done) or set the src (when it's updating). -->
+ <if expr="not chromeos">
+         <div class="icon-container"
+-            hidden="[[!shouldShowIcons_(showUpdateStatus_)]]">
++            hidden="[[shouldShowIcons_(showUpdateStatus_)]]">
+           <iron-icon
+               icon$="[[getUpdateStatusIcon_(
+                   obsoleteSystemInfo_, currentUpdateStatusEvent_)]]"
+@@ -67,14 +67,15 @@
+ </if>
+         <div class="flex cr-padded-text">
+ <if expr="not chromeos">
+-          <div id="updateStatusMessage" hidden="[[!showUpdateStatus_]]">
++          <div id="updateStatusMessage" hidden="[[showUpdateStatus_]]">
+             <div
                  inner-h-t-m-l="[[getUpdateStatusMessage_(
                      currentUpdateStatusEvent_)]]">
              </div>
-+<if expr="is_win"><a hidden$="[[!shouldShowLearnMoreLink_(currentUpdateStatusEvent_)]]" target="_blank" href="https://support.brave.com/hc/en-us/articles 360042816611-Why-isn-t-Brave-updating-automatically-on-Windows-"></if><if expr="not is_win">
-             <a hidden$="[[!shouldShowLearnMoreLink_(
+-            <a hidden$="[[!shouldShowLearnMoreLink_(
++<if expr="is_win"><a hidden$="[[shouldShowLearnMoreLink_(currentUpdateStatusEvent_)]]" target="_blank" href="https://support.brave.com/hc/en-us/articles/ 60042816611-Why-isn-t-Brave-updating-automatically-on-Windows-"></if><if expr="not is_win">
++            <a hidden$="[[shouldShowLearnMoreLink_(
                  currentUpdateStatusEvent_)]]" target="_blank"
 -                href="https://support.google.com/chrome?p=update_error">
 +                    href="https://community.brave.com?p=update_error"></if>
```

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
